src/desdemona/ui/sample-data.cljs

```diff
@@ -0,0 +1,917 @@
+    :actor-country "HK"
+    :critical true
+    :created-at "2006-06-15T04:59:05-02:00"}
+   {:id "5719f32cb968c7acaba32ef1"
+    :name "aliquip Lorem ipsum"
+    :type "Lorem_quis_exercitation"
+    :actor-username "lorrainebarker@geekola.com"
+    :actor-country "GP"
+    :critical true
+    :created-at "2000-10-25T02:08:10-02:00"}
+   {:id "5719f32c87baeb8d9d6bb741"
+    :name "officia mollit sit"
+    :type "irure_anim_reprehenderit"
+    :actor-username "howardcrane@rocklogic.com"
+    :actor-country "ZW"
+    :critical false
+    :created-at "1997-07-24T06:18:04-02:00"}
+   {:id "5719f32c392a5f66242c01c1"
+    :name "dolore veniam ipsum"
+    :type "incididunt_dolore_exercitation"
+    :actor-username "luciaconner@lotron.com"
+    :actor-country "AQ"
+    :critical true
+    :created-at "2003-04-22T11:31:55-02:00"}
+   {:id "5719f32c2334f4468fb07907"
+    :name "reprehenderit anim cillum"
+    :type "proident_elit_labore"
+    :actor-username "snyderlong@halap.com"
+    :actor-country "UZ"
+    :critical false
+    :created-at "2000-10-07T04:00:57-02:00"}
+   {:id "5719f32c956cac3d5c80b7c8"
+    :name "ipsum in aute"
+    :type "officia_elit_laborum"
+    :actor-username "kristinetanner@eweville.com"
+    :actor-country "ID"
+    :critical false
+    :created-at "2001-06-17T08:08:05-02:00"}
+   {:id "5719f32cbeb579433357d73e"
+    :name "ipsum amet eu"
+    :type "incididunt_eu_nisi"
+    :actor-username "enidcline@isbol.com"
+    :actor-country "WF"
+    :critical true
+    :created-at "1971-06-08T06:22:50-01:00"}
+   {:id "5719f32cc2c01a6297f95b06"
+    :name "nisi sint esse"
+    :type "ipsum_non_dolore"
+    :actor-username "simsboyer@geeketron.com"
+    :actor-country "NC"
+    :critical false
+    :created-at "2006-06-02T10:13:37-02:00"}
+   {:id "5719f32cfad9866d963f585d"
+    :name "amet magna do"
+    :type "consectetur_nostrud_nisi"
+    :actor-username "reynasampson@netplax.com"
+    :actor-country "GY"
+    :critical true
+    :created-at "1972-08-15T09:20:07-01:00"}
+   {:id "5719f32c144a479ce7e158db"
+    :name "velit esse id"
+    :type "irure_consectetur_dolor"
+    :actor-username "casandrahorn@austech.com"
+    :actor-country "IT"
+    :critical false
+    :created-at "1971-06-02T10:09:53-01:00"}
+   {:id "5719f32cb3782e5e2d50179c"
+    :name "et velit commodo"
+    :type "proident_occaecat_fugiat"
+    :actor-username "pearlieglass@furnigeer.com"
+    :actor-country "LI"
+    :critical false
+    :created-at "1989-08-01T05:42:38-02:00"}
+   {:id "5719f32c651aa9d805eea110"
+    :name "anim amet enim"
+    :type "tempor_tempor_dolor"
+    :actor-username "wildernavarro@extragen.com"
+    :actor-country "CU"
+    :critical false
+    :created-at "2007-07-18T04:29:26-02:00"}
+   {:id "5719f32ccb4e7dd5ffe3d1a0"
+    :name "in minim nisi"
+    :type "sint_veniam_enim"
+    :actor-username "gibsonwoodard@gink.com"
+    :actor-country "SZ"
+    :critical true
+    :created-at "2005-03-30T11:49:32-02:00"}])
```

Comment:
I generated it somewhere else and copy & pasted, that seemed a lot faster. So there's no algorithm atm